ynr/apps/bulk_adding/tests/test_bulk_add_by_party.py

```diff
@@ -102,10 +102,6 @@ def test_submit_name_for_area(self):
 
         form = response.forms[1]
 
-        # Test that the form raises and error if nothing is selected
-        response = form.submit()
-        self.assertContains(response, "This field is required.")
```

Comment:
Question: Why don't we want to test this any more?

Reply:
I've moved to defaulting to selecting "add new person" on this form, so selecting nothing is no longer an option

Reply:
Of course, the down side of this is that someone can just press next on the form, but at least this will create new people as entered, and we can always merge later.